strongbox-storage/strongbox-storage-api/strongbox/etc/conf/strongbox-authentication-providers.yaml

```diff
@@ -0,0 +1,37 @@
+strongbox:
```

Comment:
Shouldn't these files be under the `resource` path? Are they committed by mistake or they're actually needed?

Reply:
right, this was committed by mistake
thanks!